vllm/model_executor/models/llama.py

```diff
@@ -128,6 +128,8 @@ def __init__(self,
         # MistralConfig has an optional head_dim introduced by Mistral-Nemo
         self.head_dim = getattr(config, "head_dim",
                                 self.hidden_size // self.total_num_heads)
+        partial_rotary_factor = getattr(config, "partial_rotary_factor", 1)
```

Comment:
Please add a comment on this is needed by Phi

Reply:
Added a comment